packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.tsx

```diff
@@ -83,13 +78,13 @@ export class ScrollablePane extends BaseComponent<IScrollablePaneProps, {}> impl
 
     return (
       <div
-        ref='root'
+        ref={ el => this._rootRef = el! }
```

Comment:
use resolveref:
```tsx
ref={ this._resolveRef('_rootRef') }
```

Reply:
And for the other refs.

Reasoning: This is the consistent pattern we have for resolving refs, which will create a function that's cached and won't regenerate on every re-render.

Also, React 16.3 will introduce a new more formal way of creating refs, and find/replace all the `_resolveRef` instances will be easier if they're consistent :)